### PR TITLE
Excluding unlisted NuGet pacakges from version evaluation

### PR DESCRIPTION
**Pull request by fabiocav**
Initial set of changes to ignore unlisted NuGet packages from version evaluation when creating the bundle project. 

@manvkaur will follow up with additional changes for validation.

### CODE REVIEW

File: build/BuildSteps.cs [hunk 1]
```diff
@@ -101,14 +102,15 @@ public static void DownloadTemplates()
...
         public static void BuildBundleBinariesForWindows()
         {
-            Settings.WindowsBuildConfigurations.ForEach((config) => BuildExtensionsBundle(config));
+            Settings.WindowsBuildConfigurations.ForEach((config) => BuildExtensionsBundle(config).GetAwaiter().GetResult());
         }
 
         public static void BuildBundleBinariesForLinux()
         {
-            Settings.LinuxBuildConfigurations.ForEach((config) => BuildExtensionsBundle(config));
+            Settings.LinuxBuildConfigurations.ForEach((config) => BuildExtensionsBundle(config).GetAwaiter().GetResult());
```
Comment (fabiocav) [inline build/BuildSteps.cs:110]: Ideally, we'd change the orchestrator to support async flows. Not part of the scope of this PR, and this won't cause an issue for the tool.

File: build/Build.csproj [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
+	</PropertyGroup>
+	<ItemGroup>
+		<PackageReference Include="Colors.Net" Version="1.1.0" />
+		<PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+		<PackageReference Include="NuGet.Versioning" Version="6.12.1" />
+		<PackageReference Include="NuGet.Protocol" Version="6.12.1" />
+		<PackageReference Include="System.IO.Abstractions" Version="2.1.0.227" />
+		<PackageReference Include="System.Net.Http" Version="4.3.4" />
+		<PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
+		<PackageReference Include="WindowsAzure.Storage" Version="9.3.3" />
```
Comment (manvkaur) [inline build/Build.csproj:15]: @fabiocav  - Can we get remove this package - WindowsAzure.Storage? It is deprecated and I also couldn't find a reference/use of it.

Comment (manvkaur) [reply build/Build.csproj:15]: I can also upgrade System.IO.Abstractions and packages for test csproj at the sametime

Comment (manvkaur) [reply build/Build.csproj:15]: taking care of this in a separate PR.

**Review by manvkaur — APPROVED** [review]

**Review by vrdmr — APPROVED** [review]

**Pull request merged**